src/controllers/profile.ts

```diff
@@ -0,0 +1,61 @@
+import UserModel from "../models/user.model";
+import { Request, Response, NextFunction } from "express";
+
+// TODO add api doc to swagger
+
+export const getProfile = (req: Request, res: Response, next: NextFunction) => {
+  const { email } = req.body;
```

Comment:
You should first validate the input before you perform any action with it. 
Take a look in the /validator folder and write one for get and post Profile

Reply:
Added profileValidator on POST /profile, thanks I completely missed it :0 but not sure how to validate the additional info properly as it's not restricted to a certain formatting.. for now I just put it at max 256 char and imageURL is just validated as a URL but if you have better idea to validate it, please let me know!